app/policies/application_policy.rb

```diff
@@ -0,0 +1,55 @@
+# app/policies/application_policy.rb
+class ApplicationPolicy
+    attr_reader :user, :record
+  
+    def initialize(user, record)
+      @user = user
+      @record = record
+    end
+  
+    def index?
+      false
+    end
+  
+    def show?
+      scope.where(id: record.id).exists?
```

Comment:
What's the intent here?

Reply:
I think I was just trying to make sure the user can only see their own records, but yeah i see this is weird because I only did it for the show? and not for any other methods.  Also, as you mentioned above, it's probably better to just set this to false and let the reading or vital policies take care of it. I'll do that.